src/lucky/tags/base_tags.cr

```diff
@@ -75,14 +75,23 @@ module Lucky::BaseTags
     end
   end
 
+  private def build_boolean_attrs(options)
+    String.build do |attrs|
+      options.each do |value|
+        attrs << " " << LuckyInflector::Inflector.dasherize(value.to_s)
+      end
+    end
+  end
+
   private def merge_options(html_options, tag_attrs)
     options = {} of String => String
     tag_attrs.each do |key, value|
       options[key.to_s] = value
     end
 
     html_options.each do |key, value|
-      options[key.to_s] = value
+      next if key == :boolean_attrs
```

Comment:
I'd rather this key not even make it in to here, but then we would need to constantly convert NamedTuple in to hashes to reject that key while still passing the boolean_attrs around.